metricbeat/module/couchdb/server/_meta/fields.yml

```diff
@@ -0,0 +1,180 @@
+          description: >
+            Number of HTTP requests
+
+    - name: httpd_request_methods
+      type: group
+      description: >
+        HTTP request methods
+      fields:
+        - name: COPY
+          type: scaled_float
+          description: >
+            Number of HTTP COPY requests
+
+        - name: HEAD
+          type: scaled_float
+          description: >
+            Number of HTTP HEAD requests
+
+        - name: POST
+          type: scaled_float
+          description: >
+            Number of HTTP POST requests
+
+        - name: DELETE
+          type: scaled_float
+          description: >
+            Number of HTTP DELETE requests
+
+        - name: GET
+          type: scaled_float
+          description: >
+            Number of HTTP GET requests
+
+        - name: PUT
+          type: scaled_float
+          description: >
+            Number of HTTP PUT requests
+
+    - name: httpd_status_codes
+      type: group
+      description: >
+        HTTP status codes statistics
+      fields:
+        - name: "200"
+          type: scaled_float
+          description: >
+            Number of HTTP 200 OK responses
+
+        - name: "201"
+          type: scaled_float
+          description: >
+            Number of HTTP 201 Created responses
+
+        - name: "202"
+          type: scaled_float
+          description: >
+            Number of HTTP 202 Accepted responses
+
+        - name: "301"
+          type: scaled_float
+          description: >
+            Number of HTTP 301 Moved Permanently responses
+
+        - name: "304"
+          type: scaled_float
+          description: >
+            Number of HTTP 304 Not Modified responses
+
+        - name: "400"
+          type: scaled_float
+          description: >
+            Number of HTTP 400 Bad Request responses
+
+        - name: "401"
+          type: scaled_float
+          description: >
+            Number of HTTP 401 Unauthorized responses
+
+        - name: "403"
+          type: scaled_float
+          description: >
+            Number of HTTP 403 Forbidden responses
+
+        - name: "404"
+          type: scaled_float
+          description: >
+             Number of HTTP 404 Not Found responses
+
+        - name: "405"
+          type: scaled_float
+          description: >
+            Number of HTTP 405 Method Not Allowed responses
+
+        - name: "409"
+          type: scaled_float
+          description: >
+            Number of HTTP 409 Conflict responses
+
+        - name: "412"
+          type: scaled_float
+          description: >
+            Number of HTTP 412 Precondition Failed responses
+
+        - name: "500"
+          type: scaled_float
+          description: >
+            Number of HTTP 500 Internal Server Error responses
+
+    - name: couchdb 
+      type: group
+      description: >
+        couchdb statistics
+      fields:
+        - name: database_writes
+          type: scaled_float
```

Comment:
long? Same for most fields below.

Reply:
I changed all fields with float to long.